### PR TITLE
Create template from collection

### PR DESCRIPTION
**Pull request by csbailey5t**

### CODE REVIEW

File: app/views/collections/index.html.erb [hunk 1]
```diff
@@ -9,8 +9,21 @@
       <%= collection.description %>
     </div>
     <div class="col">
-      <%= link_to t('crud.edit'), edit_collection_path(collection), class: 'btn btn-secondary' %>
-      <%= link_to t('crud.destroy'), collection_path(collection), {method: :delete, remote: true, data: { confirm: t('crud.confirm') }, class: 'btn btn-danger'} %>
+      <%= form_for collection, method: :post, url: collection_templates_path do |f| %>
```
Comment (mejackreed) [inline app/views/collections/index.html.erb:12]: Is it possible to do a `form_for CollectionTemplate` here instead?

Comment (csbailey5t) [reply app/views/collections/index.html.erb:12]: Attempted, but leads to errors.

Comment (mejackreed) [reply app/views/collections/index.html.erb:12]: 😿

**Review by versae — APPROVED** [review]
👍

**Pull request merged**